.gitignore

```diff
@@ -1,5 +1,6 @@
 coverage/
 demo/dist/
+.idea/
```

Comment:
Could we maybe put this after the `.DS_Store` entry, and give that section a title like `# OS or editor files`?